Sources/HaCTML/TextNode.swift

```diff
@@ -0,0 +1,32 @@
+/**
+  Some text in an HTML tree.
+*/
+public struct TextNode {
+  let text: String
+  let escapeLevel: EscapeLevel
+  
+  /**
+    - parameters:
+      - _: the text to display
+      - escapeLevel: How much of the text to escape.
+   */
+  public init(_ text: String, escapeLevel: EscapeLevel = .all) {
+    self.text = text
+    self.escapeLevel = escapeLevel
+  }
+
+  /**
+    Enumerates how we can esape text in an HTML tree
+   */
+  public enum EscapeLevel {
+    // TODO: Come up with a solution for text in script/style tags as they are a special case for escaping.
+    /**
+      Escape everything that would otherwise render weirdly as text (e.g. < or &)
+     */
+    case all
+    /**
+      Display the text as-is. This could make the tree invalid HTML, so use at your own peril.
+     */
+    case dangerouslyRaw
```

Comment:
Nit: Might be better to call this `dangerousRaw` since otherwise it might imply there are multiple levels of 'Raw'?
(Maybe worth noting that elsewhere in the Swift API the word 'unsafe' is used, though granted that's usually for memory accesses/overflows)

Reply:
`dangerous` was inherited from React, I prefer `unsafe` because it's dirty in the same way that memory wrangling is.

I mean, the dangers are _totally_ comparable.